app/views/dashboard.erb

```diff
@@ -6,7 +6,7 @@
         <% if dashboard.current_exercises.any? %>
             <p>Keep working on these problems, or archive them.</p>
             <ul class="nav nav-stacked" id="menu-current" role="menu">
-              <% dashboard.current_exercises.limit(5).each do |exercise| %>
+              <% dashboard.current_exercises.reverse.limit(5).each do |exercise| %>
```

Comment:
I'm really curious about this line. `reverse` is a method from `Array`, when you use it the `ActiveRecord_Relation` will become an `Array`, but the `Array` class doesn't have the method `limit`, so shouldn't this raise an `NoMethodError: undefined method`?